contrib/javaee.wildfly/src/org/netbeans/modules/javaee/wildfly/config/gen/package-info.java

```diff
@@ -58,8 +58,38 @@
         docRoot="jboss-app",
         java5=true
     ),
+//    @Schema2Beans(
+//        schema="../../resources/jboss-ejb3-2_1.xsd",
+//        schemaType=SchemaType.XML_SCHEMA,
+//        outputType=OutputType.TRADITIONAL_BASEBEAN,
+//        validate=true,
+//        attrProp=true,
+//        removeUnreferencedNodes=true,
+//        docRoot="enterprise-beans",
+//        java5=true
```

Comment:
Are these comments something we could enable in the future?

Reply:
I don't think those are required at all. i should remove them